types/timed-cache/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for timed-cache 2.0
+// Project: https://github.com/HQarroum/timed-cache#readme
+// Definitions by: Dillon Sellars <https://github.com/dsellarsnr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace Cache;
+
+export = Cache;
```

Comment:
The underlying JS library is an ES module library, and it exports the `Cache` class as a default export. So instead of using `export = Cache`, you should directly export the class: `export default class Cache<T> ...`.

In the same way, you don't need the `namespace TimedCache`, and you can instead directly export the interfaces, e.g. `export interface CacheOptions`.

Reply:
Thanks @gabritto  - have updated both comments you left.